10% ru_RU translate

hided0dev opened the pull request:
[ru_RU.zip](https://github.com/DivineRPG/DivineRPG/files/3325927/ru_RU.zip)

NicosaurusRex99 commented:
I suggest you fork the repo then try again. This pr is wanting to change 250 files apparently.

Commit:
Update latest version to 1.6.3.1

Commit:
removed some testing code

Commit:
fixed grenade texture

Commit:
Readd soul -> fragment crafting recipes

AstralScourge committed:
Fix alignment of update json

Commit:
Fixed pick block not working on doors

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
Fixed pick block not working on nightmare bed

Commit:
Stopped dream lamp and infusion table from opening when sneaking and removed some print statements

Commit:
Removed unused tile entity variables

Commit:
Fixed the vethea crypts not replacing grass

Commit:
Created helper methods to streamline vethea data marker handling code

Commit:
fixed capitalisation of eden biome name

Locomenmod committed:
fixed the description of some achievements

Commit:
Fixed corrupted bullet projectile missing its texture

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
Moved twilight advancements to their own tab

Commit:
Moved iceika advancements to their own tab

Commit:
Moved the remaining advancements to their own tabs and removed nether and end entry advancements

Commit:
Stop the mod's root advancement from announcing itself

Commit:
Change unlocalized names for overworld, nether, and end advancements

Commit:
Changed unlocalized names of remaining advancements

Commit:
Stopped advancements from giving boss statues and made statue drop rate 100%

NicosaurusRex99 committed:
Create changelog.md

NicosaurusRex99 committed:
reformatted changelog

NicosaurusRex99 committed:
Added Zachtoplasm

Commit:
Added config option to disable player hats

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
Added vethea advancements

Commit:
Made the hunger's trades infinite again

Commit:
Swapped eternal archer and twilight demon drops so that halite armor is more consistently available

Locomenmod committed:
Added the key for the missing achievement, the rest are translated into Russian

Commit:
Various resprites

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
final modifier is probably not necessary

Commit:
Moved boss entities into their own package

Commit:
Moved boss loot tables to their own directory, also fixed hive queen drops

Commit:
moved twilight mobs and loot tables into their own packages

NicosaurusRex99 committed:
this looks better

Commit:
Might as well be thorough with package refactors

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
Removed always-on dev hat

Commit:
Halite armor can now be repaired with mortum chunks

Commit:
Renamed lady luna advancement

Commit:
fixed to use the new advancement name

Commit:
Remove challenge frame from some vethea advancements

Commit:
Fixed changed dimension triggers

Commit:
removed references to removed loot tables

Commit:
Made villager advancements require trades instead of just speaking to them

Commit:
quick change for clarity

Commit:
Updated version number

Commit:
Method spelling

Commit:
Remove unused file-loading method

Commit:
begin making subclasses for structures

Commit:
removed height definition from DivineLargeStructure

Commit:
Make subclass for raglok chamber

Commit:
rename structure folders

Commit:
Moved spawn altitude to constructor for flexibility

Commit:
Converted evergarden

Commit:
Converted the crypts, karos madhouse, and quadrotic post

Commit:
Converted the temples

Commit:
Converted the hive queen structure

Commit:
Converted hunger houses

Commit:
Moved chunk distance to constructor and cleaned up values

Commit:
Restored ground tree generation code

Commit:
Removed debug printouts

Locomenmod committed:
fixed translations of achievements

NicosaurusRex99 committed:
Update arcana_portal.json

Commit:
refactor overworld advancements

Commit:
refactor shark advancement and change iceika advancement to just obtaining the snowflake

Commit:
refactor twilight advancements

Commit:
refactored and renamed some arcana advancements

Commit:
Merged ayeraco horde into one advancement with placeholder name

Commit:
advancement refactoring

Commit:
finish advancement refactoring

Oshi41 committed:
Iceika speed up
Need to recheck world gen

Oshi41 committed:
Merge remote-tracking branch 'origin/1.12.2' into (origin)1.12.2

Commit:
Revert "Iceika speed up", can redo it later

This reverts commit 8a957834a9256e643a244251fd4037ecd4a32684.

Commit:
Removed lamona ability

Commit:
Made twilight boss spawners reusable

Commit:
Removed twilight boss spawner tooltips

Commit:
fixed IDs of enchantments

NicosaurusRex99 committed:
Update EntityRainbour.java

Commit:
Replace most shard textures

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
Changed name of hive queen advancement

Commit:
Changed vethea advancement icon to something not used for another advancement

Commit:
Update latest version to 1.6.4

Commit:
Update changelog

NicosaurusRex99 committed:
Update changelog.md

NicosaurusRex99 committed:
Update changelog.md

AstralScourge committed:
Update README.md

Commit:
Replace the last two shard textures

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
Revert "Revert "Iceika speed up", can redo it later"

This reverts commit fa07c4b519b1915896ff0897a8e95c1bedfebb94.

Commit:
Remove the 10x10 top block check and replaced it with a single block check

Commit:
Fix livestock hut generation

Commit:
Removed fences from livestock merchant hut

Commit:
Allowed huts to generate on flat worlds with a much lower base spawn rate

Commit:
Begin rewriting structure generation

Commit:
Continue rewriting structure generation

Commit:
Move structures to single registry class

Commit:
Mortum now has bedrock

Commit:
moved structures to structure registry, trimmed WorldGenCustomStructures

Commit:
Moved iceika houses to chunk generator

Commit:
WIP archer dungeon generation

Commit:
Added height variation

Commit:
Replaced snow globe and harp weapon textures

Commit:
Simplify component location creation

NicosaurusRex99 committed:
hats match wool colour

because vanilla changed the wool vibrance

Commit:
Implement large structure rotation

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
Implemented rollum dungeon and put dungeons into the ground

Commit:
Remove some print statements

Commit:
Implement random rotation for small structures

Commit:
Fixed placement settings of vethea structures

Commit:
Fixed missing break in switch statement causing warnings

Commit:
Fixed iceika houses and moved structure block processing methods to WorldGenUtils

Commit:
Removed structure generation print statements

Commit:
Renamed iceika chunk generator

Locomenmod committed:
localize welcome messages
fixed translations of achievements

Locomenmod committed:
Merge remote-tracking branch 'origin/1.12.2' into 1.12.2

Commit:
Readd snow to iceika

Commit:
Fixed compile error

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

NicosaurusRex99 committed:
retextured hats

and added one for people devs know

Commit:
Make vethea structures generate randomly instead of at set chunk distances

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
Tweak structure rarities a bit

Commit:
Converted hunger houses to DRPGStructureHandler

Commit:
Use getMappings instead of getAllMappings

Locomenmod committed:
added sounds to the statues

NicosaurusRex99 committed:
Compliation issue fix

NicosaurusRex99 committed:
began JER compat

NicosaurusRex99 committed:
All mobs now supported

NicosaurusRex99 committed:
updated JER

NicosaurusRex99 committed:
added the 3 seeds we can actually do

NicosaurusRex99 committed:
Update changelog.md

Commit:
Fix cascading worldgen by adding size values to positions of rotated structures

Commit:
Temporarily disable most iceika generation to debug cascading trees

Commit:
Removed now-unused legacy structure code

Commit:
Removed programatic iceika gen stuff since we're using structure block nbt now

Commit:
Removed programmatic iceika gen stuff since we're using structure block nbt now

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
Changed iceika tree generation algorithm and made them slightly less common

Commit:
Fixed cascading iceika trees

NicosaurusRex99 committed:
Update EntityDivineFlyingMob.java

NicosaurusRex99 committed:
Update changelog.md

Locomenmod committed:
Update ru_ru.lang

Commit:
Internal tar rename

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
Revert tar rename for testing purposes

This reverts commit 7376dce6a1b5ed4b9c622d33d373be505f62fbad.

Commit:
Revert "Revert tar rename for testing purposes"

This reverts commit a2507fd9b96664ee4d1a854e22c64978d812bbb3.

Commit:
Renamed tar to smoldering tar

Commit:
Uncommented iceika generation code and gave it back its skylight

Commit:
Remove debug print statement

Commit:
Remove the skylight again because it didn't seem to change anything

Commit:
Update to 1.6.4.1

Commit:
Remove unused methods from JER compat class

Commit:
Some code cleanup

Commit:
Made twilight boss spawners require an item

NicosaurusRex99 committed:
mobs move around but still float up

NicosaurusRex99 committed:
tweaks to aquatic mobs

NicosaurusRex99 committed:
Update changelog.md

Commit:
Refactor creative tab variable names

NicosaurusRex99 committed:
Update changelog.md

NicosaurusRex99 committed:
more particles

Commit:
Added 16x16 dungeon components for arcana bosses

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
Make whale streams more spread out

Commit:
Update latest version to 1.6.4.1

Commit:
Update changelog with 1.6.4.1 changes

AstralScourge committed:
Update changelog.md

AstralScourge committed:
Update changelog.md

Locomenmod committed:
localize "BlockSingleUseSpawner"

FrozenYT committed:
Error correction + Add "message.friend="

FrozenYT committed:
Update Event Login.

EN: Added message: message to enter if you are a friend of the project.


RU: Добавлено сообщение для тех, кто находится в списке друзей у проекта.

Locomenmod committed:
Update EventClientLogin.java

FrozenYT committed:
Update en_us.lang

Add "message.friend"

Locomenmod committed:
Merge pull request #147 from FrozenYT/patch-2

Update EventClientLogin

Locomenmod committed:
Merge pull request #146 from FrozenYT/patch-1

Error correction + Add "message.friend="

Commit:
Fixed arcana portal room generating in the middle of four other rooms

NicosaurusRex99 committed:
Update en_us.lang

NicosaurusRex99 committed:
added nuggets

Oshi41 committed:
Arcana boss rooms

Commit:
Maybe fix the define id issue

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
Remove many tooltips with redundant information for now

Commit:
Readd infinite usage tooltips and add instant consumption tooltip for magic meat

Commit:
Fixed swapped dark/normal degraded brick textures

Commit:
Fixed npe with attack target

NicosaurusRex99 committed:
added frosted alure

FrozenYT committed:
Error correction

Locomenmod committed:
Merge pull request #149 from FrozenYT/patch-3

Error correction

Commit:
Change slab system to not use wooden plank types and added a sample degraded brick slab

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
cleanup

NicosaurusRex99 committed:
tweaks to pumpkin pie

NicosaurusRex99 committed:
hide arcana if full config added

Commit:
Added ancient brick slabs and ancient/degraded brick stairs

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
Update lang file

Commit:
Add stone wall class

NicosaurusRex99 committed:
Update mcmod.info

Commit:
Arcana block resprites

Commit:
New description

Commit:
Add degraded brick wall

Commit:
Made iceika dungeon blocks breakable and added a crafting recipe for icy bricks

Commit:
Fixed top slabs becoming bottom slabs on world reload

Commit:
Made arcana plants normal farmland plants

NicosaurusRex99 committed:
Update JERCompat.java

Locomenmod committed:
fix ru lang
added baubles

Locomenmod committed:
added baubles support for vethea

Locomenmod committed:
added functional for miners amulet

Locomenmod committed:
Update BaublesCompat

NicosaurusRex99 committed:
we also want the amulet to work without baubles mod

NicosaurusRex99 committed:
Added sit to tamed mobs

NicosaurusRex99 committed:
added missing nugget

NicosaurusRex99 committed:
Update en_us.lang

NicosaurusRex99 committed:
Update changelog.md

NicosaurusRex99 committed:
Update changelog.md

NicosaurusRex99 committed:
Update changelog.md

NicosaurusRex99 committed:
ender armor negates ender pearl damage

NicosaurusRex99 committed:
Update changelog.md

NicosaurusRex99 committed:
Wither armor now withers attacker

NicosaurusRex99 committed:
Update changelog.md

FishInMars committed:
Update zh_cn.lang for 1.12.2 and a small request

This is an update for zh_cn.lang (1.12.2).  And I'm wondering if I can repost DivineRPG to a Chinese forum called MCBBS, you know, because of GFW , it's hard for Chinese players to know many other mods. And I will not use is for profit and I will link to curseforge pages.I will keep everything you have asked.

NicosaurusRex99 committed:
Merge pull request #150 from Imbarainbow/1.12.2 (Simplified Chinese Localization)

Update zh_cn.lang for 1.12.2 and a small request

NicosaurusRex99 committed:
Update changelog.md

Commit:
Implemented arcana maze generation using sample pieces

Commit:
Soul sludge retexture

Commit:
Replaced all magic constant 64 references with maze size constant

NicosaurusRex99 committed:
jei compat to arcanium ore

Commit:
Some further cleanup

Commit:
Update door block textures

Commit:
Merge branch '1.12.2' of https://github.com/NicosaurusRex99/DivineRPG into 1.12.2

Commit:
Updated some textures and made arcanium power give off light

NicosaurusRex99 committed:
Update changelog.md

NicosaurusRex99 committed:
added basic arcana rooms

NicosaurusRex99 committed:
added hallway variant

NicosaurusRex99 committed:
made arcana rooms frosted charge compatible

Commit:
Trim some fat from the arcana chunk generator

Commit:
Remove old arcana components

Commit:
Merge some testing code

Commit:
enabled easier switching between test maze pieces and actual rooms

Commit:
fixed some comments

Commit:
testing

Commit:
Revert "testing"

This reverts commit a4cd97892692cda2bcf4bf465154a1378bf6664d.

Commit:
Fixed axe speeds

AstralScourge committed:
Merge branch '1.12.2' into 1.12.2
